port_ocean/log/sensetive.py

```diff
@@ -0,0 +1,74 @@
+import re
+from typing import Callable, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from loguru import Record
+
+# https://github.com/h33tlit/secret-regex-list
+secret_patterns = {
+    "Cloudinary": r"cloudinary:\/\/.*",
```

Comment:
super cool, I am thinking maybe we would like to reduce the amount of patterns to those logically related to what ocean will support exporting. 

e.g. square, twitter sounds too far from integration that will be supported